os/mounting-storage.md

```diff
@@ -1,6 +1,6 @@
 # Mounting Storage
 
-Many platforms provide attached storage, but it must be mounted for you to take advantage of it. You can easily do this via [cloud-config]({{site.baseurl}}/docs/cluster-management/setup/cloudinit-cloud-config) with a `.mount` unit. Here's an example that mounts an [EC2 ephemeral disk]({{site.baseurl}}/docs/running-coreos/cloud-providers/ec2/#instance-storage):
+Many platforms provide attached storage, but it must be mounted for you to take advantage of it. You can easily do this via [Cloud-Config][cloud-config] with a `.mount` unit. Here's an example that mounts an [EC2 ephemeral disk][ec2-instance-storage]:
```

Comment:
the `[cloud-config]` references the link defined on line 23, so this should work just fine. As an aside, we don't really have a style guide for this type of thing, but defining it this way means you can reference the link as many times as you want, but update the location with just one change.